components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -984,6 +1001,15 @@ public void revokeAccessTokensIndividually(String[] tokens) throws IdentityOAuth
                 OAuth2TokenUtil.postUpdateAccessTokens(accessTokenId, OAuthConstants.TokenStates.
                         TOKEN_STATE_REVOKED);
             }
+
+            if (isTokenCleanupFeatureEnabled) {
+                for (String token : tokens) {
+                    oldTokenCleanupObject.cleanupTokenByTokenValue(
+                            getHashingPersistenceProcessor().getProcessedAccessTokenIdentifier(token), connection);
+                }
+            }
+            connection.commit();
```

Comment:
we should remove the commit in #998 right?

Reply:
yes with  the revamp process "
Move JDBC transaction into structured UnitOfWork design pattern" connection.commit(); has removed